guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -2,11 +2,15 @@
 = Disabling DNS, DHCP, and TFTP for unmanaged networks
 
 If you want to manage TFTP, DHCP, and DNS services manually, you must prevent {Project} from maintaining these services on the operating system and disable orchestration to avoid DHCP and DNS validation errors.
-However, {Project} does not remove the back-end services on the operating system.
++
+[IMPORTANT]
+====
+Disabling these proxy features means {Project} will no longer orchestrate DNS, DHCP, and TFTP, but it does stop or remove the corresponding services.
+====
 
 .Procedure
 
-. On {ProjectServer}, enter the following command:
+. On {ProjectServer}, enter the following command to disable the Foreman Proxy features for DHCP, DNS, and TFTP:
```

Comment:
```suggestion
. Disable DHCP, DNS, and TFTP integration on your {ProjectServer}:
```

suggestion (non-blocking): Reword to avoid "enter the following command" and "Foreman Proxy".

Alternative to "integration": "Disable the X features on"